tests/unit/blog-utils/TestStaticAddonCard.js

```diff
@@ -128,15 +139,28 @@ describe(__filename, () => {
     expect(root.find(Rating)).toHaveProp('styleSize', 'small');
   });
 
-  it('hides ratings when there is no data', () => {
+  it('shows 0 ratings when there is no data', () => {
     const addon = createInternalAddonWithLang({
       ...fakeAddon,
       ratings: null,
```

Comment:
ah, that's why the tests didn't pick it up previously - the api always returns an object for ratings, and always returns numbers for the values.

Reply:
ok, I updated the patch accordingly.